README.md

```diff
@@ -52,6 +52,29 @@ following snippet to your config:
 (vhdl-ext-lsp-set-server 've-rust-hdl)   ; `lsp' config
 ```
 
+## Use in Neovim
+
+Neovim provides an LSP client to the VHDL_LS language server. Install the VHDL_LS  
+binary e.g. via Cargo. The binary must be on the path and executable (if you  
+can run "vhdl_ls -h" in the terminal then you're good). Put the "vhdl_libraries"  
+directory into the parent directory of the VHDL_LS binary. 
+
+In your Neovim config.lua add the following:  
+
+```lua
+function STARTVHDLLS()
+  vim.lsp.start({
+    name = 'vhdl_ls',
+    cmd = {'vhdl_ls'},
+  })
+end
+vim.api.nvim_set_keymap('n', '<F5>', ':lua STARTVHDLLS()<CR>', { noremap = true, silent = true })
+```
+
+This is just one example, there are many ways to run the language server. Anyway  
```

Comment:
What are the other ways to run the language server?

Reply:
What I meant is that you can also autorun the LS, or start it when you enter a buffer, or tie it to another plugin somehow. Essentially any way you like within what Neovim allows. But wanted to keep the "How to" short. Should I just remove this remark?

Reply:
I like it, but maybe rephrase it to something like "Using the example above, pressing F5 while inside NeoVim starts the language server. There are also other options, like automatically starting it when opening a certain file type, see the doc for more..."